ffi/util.lua

```diff
@@ -60,10 +60,27 @@ int WideCharToMultiByte(
 );
 ]]
 
+local getlibprefix = function()
+    -- Apple M1 homebrew installs libraries outside of default searchpaths,
+    -- and dyld environment variables are sip-protected on MacOS, cf. https://github.com/Homebrew/brew/issues/13481#issuecomment-1181592842
+    local libprefix = os.getenv("KO_DYLD_PREFIX")
+
+    if not libprefix then
+        local std_out = io.popen("brew --prefix", "r")
+            if std_out then
+                libprefix = std_out:read("*line")
+                std_out:close()
+            end
+    end
+
+    return libprefix
+end
+
 require("ffi/posix_h")
 
 local util = {}
 
+util.KO_DYLD_PREFIX = getlibprefix()
```

Comment:
This shouldn't be called except on OS X, so probably on line 654 below.

Reply:
Yeah, thanks for the unbreaking commit elsewhere, I didn't consider this might throw an exception ;)

Reply:
Good thing the CI caught it. ;-)